### DISCUSSION TITLE
Hardbreak and Softbreak

### DISCUSSION THREAD

**Discussion by layan-khater** [markdoc/markdoc#171] [Q&A] [answered]
Hi there!

I saw that there is a soft and hard break nodes, can what they are for and do they have any relation to HTML line break?

**Comment by mfix-stripe** [comment, marked as answer]
Hey @layan-khater, this spec describes how they are similar:
* https://spec.commonmark.org/0.30/#hard-line-breaks
* https://spec.commonmark.org/0.30/#soft-line-breaks
